interfaces/builtin/system_observe.go

```diff
@@ -123,6 +123,9 @@ ptrace (read),
 /sys/fs/cgroup/cpu,cpuacct/cpu.stat r,
 /sys/fs/cgroup/memory/memory.stat r,
 
+# Allow reading the system max CPU resource constraints 
+/sys/fs/cgroup/system.slice/cpu.max r,
```

Comment:
are you sure this is want this, which limits how much cpu time a process can consume, rather than /sys/devices/system/cpu ?